example3/OpenTelemetry.Metric.Sdk/Exporter.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Concurrent;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Diagnostics.Metric;
+
+namespace OpenTelemetry.Metric.Sdk
+{
+    public abstract class Exporter
+    {
+        public abstract void Export(ExportItem[] exports);
+        public abstract void BeginFlush();
+
+        public abstract void Start(CancellationToken token);
+
+        public abstract void Stop();
+    }
+
+    public class ExportItem

```

Comment:
is this something which will evolve to match the OTLP protos?

Reply:
Just a placeholder to show possible end-to-end.  But, yes, when we're done, this will be evolved or replaced with correct OTLP proto.